openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -1110,7 +1112,13 @@ List<EntityRelationshipObject> findFromBatch(
         @BindList("toIds") List<String> toIds, @Bind("relation") int relation);
 
     @SqlQuery(
-        "SELECT fromId, toId, fromEntity, toEntity, relation "
+        "SELECT * FROM ( SELECT *, ROW_NUMBER() OVER (ORDER BY fromId) AS row_num FROM entity_relationship WHERE relation = :relation) AS rec WHERE row_num > :after ORDER BY row_num LIMIT :limit;")
```

Comment:
any reason to change this query

Reply:
i added a new one, but that too i am not user , removing it